diff --git a/Makefile b/Makefile
index 3a13374..61b10d2 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CC = gcc
-LIBS = -lncursesw
+LIBS = -lncurses
 
 OBJS = main.o shapes.o
 donut : $(OBJS)
diff --git a/main.c b/main.c
index f936ebf..198c280 100644
--- a/main.c
+++ b/main.c
@@ -104,6 +104,12 @@ int main(void) {
     noecho();
     curs_set(0); 
     timeout(0);
+    start_color();
+    init_pair(COLOR_YELLOW, COLOR_YELLOW, COLOR_BLACK);
+    init_pair(COLOR_RED, COLOR_RED, COLOR_BLACK);
+    init_pair(COLOR_MAGENTA, COLOR_MAGENTA, COLOR_BLACK);
+    init_pair(COLOR_CYAN, COLOR_CYAN, COLOR_BLACK);
+    init_pair(COLOR_BLUE, COLOR_BLUE, COLOR_BLACK);
     
     signal(SIGINT, sig_handler);
     signal(SIGQUIT, sig_handler);
diff --git a/shapes.c b/shapes.c
index f4dec30..51eebe9 100644
--- a/shapes.c
+++ b/shapes.c
@@ -23,17 +23,18 @@ typedef struct Trig {
   x *= _; \
   y *= _;
 
-void draw_char(uint8_t *output, char *emoji, WINDOW *win) {
+void draw_char(uint8_t *output, int color, WINDOW *win) {
+    wattrset(win, COLOR_PAIR(color));
     wmove(win, 0, 0);
     for(int k = 0; k < total; k++) {
         if (TESTBIT(output, k)) {
-            waddstr(win, emoji);
-            k++; //depends on how many 'cursor' places the symbol takes
+	    waddch(win, '#');
         } else {
-            waddch(win, 32);// ascii space
+            waddch(win, ' ');
         }
     }
     wrefresh(win);
+    wattroff(win, COLOR_PAIR(color));
 }
 
 void term_xy(float x, float y, uint8_t *output) {
@@ -67,7 +68,7 @@ void draw_donut(Donut *donut, WINDOW *win, uint8_t *output) {
 	}
 	R(0.07, costheta, sintheta)
     }
-    draw_char(output, "🍩", win);
+    draw_char(output, COLOR_YELLOW, win);
     R(0.07, donut->cosx, donut->sinx)
     R(0.03, donut->cosz, donut->sinz)
 }
@@ -100,7 +101,7 @@ void draw_heart(Heart *heart, WINDOW *win, uint8_t *output) {
 	R(0.02, cosu, sinu)
 	R(0.06, cos3u, sin3u)
     }
-    draw_char(output, "💞", win);
+    draw_char(output, COLOR_RED, win);
     R(0.07, heart->cosx, heart->sinx)
     R(-0.03, heart->cosz, heart->sinz);
 }
@@ -133,7 +134,7 @@ void draw_cube(Cube *cube, WINDOW *win, uint8_t *output) {
           cube_store(output, x, -z, half, &trig);
         }
     }
-    draw_char(output, "🧊", win);
+    draw_char(output, COLOR_BLUE, win);
     R(-0.07, cube->cosx, cube->sinx)
     R(0.03, cube->cosz, cube->sinz)
 }
@@ -199,7 +200,7 @@ void draw_knot(Knot *knot, WINDOW *win, uint8_t *output) {
         }
 	R(0.02, cosu, sinu)
     }
-    draw_char(output, "🌺", win);
+    draw_char(output, COLOR_MAGENTA, win);
     R(0.03, knot->cosy, knot->siny)
 }
 
@@ -224,7 +225,7 @@ void draw_cone(Cone *cone, WINDOW *win, uint8_t *output) {
 	}
 	R(0.02, cosu, sinu)
     }
-    draw_char(output, "🥦", win);
+    draw_char(output, COLOR_CYAN, win);
     R(-0.07, cone->cosx, cone->sinx)
     R(0.03, cone->cosz, cone->sinz)
 }
